packages/rn-tester/js/utils/RNTesterList.ios.js

```diff
@@ -60,11 +55,6 @@ const ComponentExamples: Array<RNTesterExample> = [
     module: require('../examples/Layout/LayoutEventsExample'),
     supportsTVOS: true,
   },
-  {
-    key: 'MaskedViewExample',
```

Comment:
Did you delete the MaskedViewExample files?

Reply:
@anku255 Yeah i think i forgot to delete those files, doing that.